mysql-test/suite/rocksdb/t/rpl_statement_not_found.test

```diff
@@ -1,2 +1,4 @@
---source rpl_row_not_found.inc
+--source include/have_binlog_format_statement.inc
```

Comment:
Slightly misleading testcase name: should be something like rpl_stm_row_not_found.test